Update enterprise permissions documentation (#9813)
fishtown-analytics/docs.getdbt.com: website/docs/docs/platform/manage-access/enterprise-permissions.md

Notable features: 
- Database admin is a project-level set. 
- Can set up and maintain environment variables and <Constant name="semantic_layer" /> configs.

> - Write access to data platform configurations within environments (credentials, warehouse, schema per environment) they have access to, including:

Reviewer: "They have access to" is a little ambiguous here. It's not clear whether it's referring to environments, data platform configurations, or the fields in brackets (credentials, warehouse, schema).

Since the nested bullets use two different scopes, maybe make that more explicit?

> - Write access to data platform configurations within environments (credentials, warehouse, schema per environment) they have access to, including:

Author: ```suggestion
- Write access to data platform configurations within environments (credentials, warehouse, schema per environment), including:
```

>   - Editing profile configs for environments with write access (profile name, deployment credentials, schema, role, database, warehouse, and extended attributes)

Reviewer: L147: The parenthetical here overlaps with the nested list on L148, which already covers deployment credentials, warehouse, and schema.

Maybe remove the parenthetical and let the nested list provide the detail, or keep the parenthetical and shorten L148 so it's not repeating the same information?

>   - Editing profile configs for environments with write access (profile name, deployment credentials, schema, role, database, warehouse, and extended attributes)

Reviewer: ```suggestion
  - Editing [profile](/docs/platform/about-profiles) configs for environments with write access (profile name, deployment credentials, schema, role, database, warehouse, and extended attributes)
```

>   - Editing [profile](/docs/platform/about-profiles) configs for environments they have write access to (profile name, deployment credentials, schema, role, database, warehouse, and extended attributes)

Author: ```suggestion
  - Editing [profile](/docs/platform/about-profiles) configs (such as profile name, deployment credentials, connection overrides like schema, role, database, warehouse, and extended attributes)
```

>   - Editing [profile](/docs/platform/about-profiles) configs for environments with write access (profile name, deployment credentials, schema, role, database, warehouse, and extended attributes)

Author: ```suggestion
  - Editing [profile](/docs/platform/about-profiles) configs for environments they have write access to (profile name, deployment credentials, schema, role, database, warehouse, and extended attributes)
```

>   - Editing [profile](/docs/platform/about-profiles) configs like profile name, deployment credentials, extended attributes, and connection overrides such as schema, role, database, and warehouse (fields vary by data platform).

Author: ```suggestion
  - Editing [profile](/docs/platform/about-profiles) configs like profile name, deployment credentials, extended attributes, and connection overrides such as `schema`, `role`, `database`, and so on (fields vary by data platform).
```

>   - Creating new profiles for projects they have access to, including setting which connection the profile is assocaited with

Reviewer: ```suggestion
  - Creating new profiles for projects they have access to, including setting which connection the profile is associated with
```

>   - Creating new profiles for projects they have access to, including setting which connection the profile is assocaited with

Reviewer: Lines 147 - 149 - The scope feels a little inconsistent here. One bullet says "they have access to", while the nested bullets refer to "environments with write access" and "projects they have access to". Maybe align the wording, or make it explicit if creating and editing are intentionally scoped differently?

>   - Creating new profiles for projects they have access to, including setting which connection the profile is associated with

Reviewer: I checked the `dbt-cloud` repo, and I don't think these two statements quite match what's there today.

Database Admin has `profiles_write` and `environments_read`, not `environments_write`. From what I can see, profile write also isn't gated on environment write access: [permission set](https://github.com/dbt-labs/dbt-cloud/blob/d2f885a62dc2ed60f346cdd644971c21417a979a/sinter/models/permissions/sets/project/database_admin.py#L31-L55), [profile permissions](https://github.com/dbt-labs/dbt-cloud/blob/d2f885a62dc2ed60f346cdd644971c21417a979a/sinter/services/profile/profile_permission.py#L5-L14).

I also noticed that a profile consists of the name, connection, credentials, and extended attributes. Schema, role, database, and warehouse are Snowflake credential fields rather than profile fields: [profile model](https://github.com/dbt-labs/dbt-cloud/blob/d2f885a62dc2ed60f346cdd644971c21417a979a/sinter/database/profile.py#L30-L75), [Snowflake credentials](https://github.com/dbt-labs/dbt-cloud/blob/d2f885a62dc2ed60f346cdd644971c21417a979a/sinter/database/credentials/snowflake.py#L60-L64).

Could we list those four profile fields instead and remove the environment write-access wording?

>   - Creating new profiles for projects they have access to, including setting which connection the profile is associated with

Reviewer: I checked the `dbt-cloud` repo, and I don't think these two statements quite match what's there today.

Database Admin has `profiles_write` and `environments_read`, not `environments_write`. From what I can see, profile write also isn't gated on environment write access: [permission set](https://github.com/dbt-labs/dbt-cloud/blob/d2f885a62dc2ed60f346cdd644971c21417a979a/sinter/models/permissions/sets/project/database_admin.py#L31-L55), [profile permissions](https://github.com/dbt-labs/dbt-cloud/blob/d2f885a62dc2ed60f346cdd644971c21417a979a/sinter/services/profile/profile_permission.py#L5-L14).

I also noticed that a profile consists of the name, connection, credentials, and extended attributes. Schema, role, database, and warehouse are Snowflake credential fields rather than profile fields: [profile model](https://github.com/dbt-labs/dbt-cloud/blob/d2f885a62dc2ed60f346cdd644971c21417a979a/sinter/database/profile.py#L30-L75), [Snowflake credentials](https://github.com/dbt-labs/dbt-cloud/blob/d2f885a62dc2ed60f346cdd644971c21417a979a/sinter/database/credentials/snowflake.py#L60-L64).

Double checked with Fidgeotto / Fidget [here](https://dbt-labs.slack.com/archives/C0BLQ2HB6TC/p1786964059185709)

Could we list those four profile fields instead and remove the environment write-access wording?

Author: thanks @nataliefiann and appreciate the info!

i'll push back on saying that profile only consists of name/connection/credentials/extended attributes. teradata/snowflake/fabric all have schema (for example) and from a user perspective they're all editable within the profile form (see screenshot). so removing them from the docs entirely would be wrong and i dont think we should remove them, esp as teh original ask was from a user asking for the permission doc to be more detailed. this is removing those details they suggested would be helpful for them to easily grasp what the permission can do at a granular level. 

<img width="890" height="1370" alt="Screenshot 2026-08-17 at 12 08 04" src="https://github.com/user-attachments/assets/233fd8d3-0f73-4f31-9a8b-ebee1d5025c0" />
<img width="586" height="1281" alt="Screenshot 2026-08-17 at 12 08 36" src="https://github.com/user-attachments/assets/8bb7eed2-3759-4e6a-b7d6-385cf9dcfd4f" />
